devops/scripts/benchmarks/benches/compute.py

```diff
@@ -334,6 +335,16 @@ def name(self):
 
         return f"api_overhead_benchmark_{self.runtime.value} SubmitKernel {order}{completion_str}{events_str}"
 
+    def display_name(self) -> str:
```

Comment:
can you add "NumKernels" to the display name?

Reply:
Done